macOS Monterey: dyld[87026]: Symbol not found: (_vidattr$NCURSES60)   Referenced from: '/private/var/folders/.../T/MariaDB4j/base/bin/mariadb'   Expected in: '/usr/lib/libncurses.5.4.dylib'

https://github.com/MariaDB4j/MariaDB4j/pull/1124#issuecomment-2770477126 :

```
[CommonsExecStreamPumper-pool-20-thread-3] INFO ch.vorburger.exec.ManagedProcess - mariadb:   Referenced from: '/private/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base/bin/mariadb'
[CommonsExecStreamPumper-pool-20-thread-3] INFO ch.vorburger.exec.ManagedProcess - mariadb:   Expected in: '/usr/lib/libncurses.5.4.dylib'
[CommonsExecDefaultExecutorpool-17-thread-1] ERROR ch.vorburger.exec.ManagedProcess - Program [/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base/bin/mariadb, --default-character-set=utf8, --socket=/private/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j.64117.sock] (in working directory /var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base) failed unexpectedly
org.apache.commons.exec.ExecuteException: Process exited with an error: 134 (Exit value: 134)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:355)
	at org.apache.commons.exec.DefaultExecutor.lambda$execute$0(DefaultExecutor.java:269)
	at java.base/java.lang.Thread.run(Thread.java:840)
[main] ERROR ch.vorburger.exec.ManagedProcess - Program [/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base/bin/mariadb, --default-character-set=utf8, --socket=/private/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j.64117.sock] (in working directory /var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base) failed
java.util.concurrent.ExecutionException: org.apache.commons.exec.ExecuteException: Process exited with an error: 134 (Exit value: 134)
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073)
	at ch.vorburger.exec.ManagedProcess.waitForExitMaxMsWithoutLog(ManagedProcess.java:481)
	at ch.vorburger.exec.ManagedProcess.waitForExit(ManagedProcess.java:455)
	at ch.vorburger.mariadb4j.DB.run(DB.java:492)
	at ch.vorburger.mariadb4j.DB.run(DB.java:444)
	at ch.vorburger.mariadb4j.DB.run(DB.java:376)
	at ch.vorburger.mariadb4j.DB.run(DB.java:386)
	at ch.vorburger.mariadb4j.DB.createDB(DB.java:507)
	at ch.vorburger.mariadb4j.junit.MariaDB4jRule.initDB(MariaDB4jRule.java:56)
	at ch.vorburger.mariadb4j.junit.MariaDB4jRule.before(MariaDB4jRule.java:51)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:50)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 134 (Exit value: 134)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:355)
	at org.apache.commons.exec.DefaultExecutor.lambda$execute$0(DefaultExecutor.java:269)
	at java.base/java.lang.Thread.run(Thread.java:840)
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 20.699 sec <<< FAILURE!
validateSourceInitialization(ch.vorburger.mariadb4j.tests.junit.MariaDB4jUnitRuleInitDBTest)  Time elapsed: 20.697 sec  <<< ERROR!
ch.vorburger.exec.ManagedProcessException: An error occurred while running a command: create database if not exists `junittest`;
	at ch.vorburger.mariadb4j.DB.run(DB.java:494)
	at ch.vorburger.mariadb4j.DB.run(DB.java:444)
	at ch.vorburger.mariadb4j.DB.run(DB.java:376)
	at ch.vorburger.mariadb4j.DB.run(DB.java:386)
	at ch.vorburger.mariadb4j.DB.createDB(DB.java:507)
	at ch.vorburger.mariadb4j.junit.MariaDB4jRule.initDB(MariaDB4jRule.java:56)
	at ch.vorburger.mariadb4j.junit.MariaDB4jRule.before(MariaDB4jRule.java:51)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:50)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: ch.vorburger.exec.ManagedProcessException: Program [/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base/bin/mariadb, --default-character-set=utf8, --socket=/private/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j.64117.sock] (in working directory /var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base) failed with Exception: , last 100 lines of console:
dyld[87026]: Symbol not found: (_vidattr$NCURSES60)
  Referenced from: '/private/var/folders/38/70vr92w528b6f4zr0wxvd1600000gn/T/MariaDB4j/base/bin/mariadb'
  Expected in: '/usr/lib/libncurses.5.4.dylib'
	at ch.vorburger.exec.ManagedProcess.waitForExitMaxMsWithoutLog(ManagedProcess.java:494)
	at ch.vorburger.exec.ManagedProcess.waitForExit(ManagedProcess.java:455)
	at ch.vorburger.mariadb4j.DB.run(DB.java:492)
	... 31 more
Caused by: java.util.concurrent.ExecutionException: org.apache.commons.exec.ExecuteException: Process exited with an error: 134 (Exit value: 134)
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073)
	at ch.vorburger.exec.ManagedProcess.waitForExitMaxMsWithoutLog(ManagedProcess.java:481)
	... 33 more
Caused by: org.apache.commons.exec.ExecuteException: Process exited with an error: 134 (Exit value: 134)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:355)
	at org.apache.commons.exec.DefaultExecutor.lambda$execute$0(DefaultExecutor.java:269)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

--- a/README.md
+++ b/README.md
@@ -292,6 +292,9 @@ A: You need a more recent Java version than the one you are running on to be abl
 Q: `dyld[23092]: Library not loaded: /opt/homebrew/opt/openssl@3/lib/libssl.3.dylib Referenced from: <F56C2AF5-D763-3960-A454-40591B10F714> /Users/flow/MariaDB4j/mariaDB4j/target/MariaDB4jSpringServiceOverrideBySpringValueTest/baseDir/bin/mariadbd Reason: tried: '/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib' (no such file), '/opt/homebrew/opt/openssl@3/lib/libssl.3.dylib' (no such file)`
 A: This can happen on MacOS, and can be fixed by [installing Homebrew](https://brew.sh), and then (one time) doing `brew install openssl@3`. (See [issue #497](https://github.com/MariaDB4j/MariaDB4j/issues/497#issuecomment-2762820549) for technical background.)
 
+Q: `dyld[87026]: Symbol not found: (_vidattr$NCURSES60) Referenced from: '/private/var/folders/.../T/MariaDB4j/base/bin/mariadb'`
+A: This is a macOS Monterey specific issue. Please upgrade to a more recent version of macOS (like Sequoia v15.4, or later). See [Issue #1149](https://github.com/MariaDB4j/MariaDB4j/issues/1149) for background.
+
 Q: `ERROR ch.vorburger.exec.ManagedProcess - mysql: /tmp/MariaDB4j/base/bin/mysql: error while loading shared libraries: libncurses.so.5: cannot open shared object file: No such file or directory`
 A: This could happen e.g. on Fedora 24 if you have not previous installed any other software package which requires libncurses, and can be fixed by finding the RPM package which provides `libncurses.so.5` via `sudo dnf provides libncurses.so.5` and then install that via `sudo dnf install ncurses-compat-libs`. On Ubuntu Focal 20.04, you need to `sudo apt update && sudo apt install libncurses5`.
 (This is fixed if you use the 11.4.5 instead of 10.11.5 MariaDB binaries, which [are released](https://github.com/MariaDB4j/MariaDB4j/issues/1137) with MariaDB4j v3.2.0.)

